cylc/flow/network/schema.py

```diff
@@ -1514,23 +1514,33 @@ class Arguments:
     result = GenericScalar()
 
 
-class Hold(Mutation):
+class SetHoldPoint(Mutation):
     class Meta:
         description = sstrip('''
-            Hold a workflow or tasks within it.
+            Set workflow hold after cycle point. All tasks after this point
+            will be held.
         ''')
-        resolver = partial(mutator, command='hold')
+        resolver = partial(mutator, command='set_hold_point')
 
     class Arguments:
         workflows = List(WorkflowID, required=True)
-        tasks = List(
-            NamespaceIDGlob,
-            description='Hold the specified tasks rather than the workflow.'
+        point = TimePoint(
```

Comment:
There is a special type for cycle points:

```suggestion
        point = CyclePoint(
```

Under the hood these types are all just strings but the types allow the UI to work out what "things" a mutation works with.